### ISSUE TITLE
Simultaneously color reads by YC and MM/ML tag

### ISSUE THREAD

**Issue by pre-mRNA** [igvteam/igv#1403]
Would this be possible? The YC tag colors the entire alignment, whereas the MM/ML tag only typically color individual bases. 
It would be great to be able to visualise both simultaneously.

**Comment by jrobinso** [comment]
I don't know, have you tried it?

**Comment by pre-mRNA** [comment]
Right now, the color modes are mutually exclusive, but it should be possible to show both at the same time, using YC tag as the base alignment color and MM/ML tags for individual modified bases.

I guess this is more of a feature request than an issue.

**Comment by jrobinso** [comment]
Could you prepare a small dataset with "YC"  and "MM/ML" tags?  Just a few reads.  I agree you should be able to do both.  I'm surprised it doesn't color alignments by "YC" by default, in which case you could select base modifications from the menu.

**Comment by pre-mRNA** [comment]
Hey, sorry for the delay on my end. I've uploaded a test datasets of reads annotated with YC and MM/ML tags, mapped to HG38. 

The bam file is attached: 

[HNRNPA1.zip](https://github.com/igvteam/igv/files/15257005/HNRNPA1.zip)

For reference, here are the views I get when I color by YC tag, and later by `base modificiation (m6A). I'm using IGV 2.17.3
It would be great to be able to see both views, with the YC tag on the bottom, and the modification calls overlaid atop this information. 

![YC tags view](https://github.com/igvteam/igv/assets/88181189/d44f1df2-8f74-4b26-b888-1e2619f18adb)
![MM tags view](https://github.com/igvteam/igv/assets/88181189/56450d77-c080-441d-b106-1dc1d6eae571)

**Comment by pre-mRNA** [comment]
Bumping

**Comment by jrobinso** [comment]
@lbergelson It occurs to me that we have conflated 2 things here.  Color by tag applies to the alignments,  color by modifications is really how to color individual bases.   Coloring mismatches, and bisulfite seq options as well.  Perhaps we can split these (alignment and base color) as separate options.  Its not trivial to do this and maintain the same behavior by default, and importantly the same look for the many saved sessions out there.

**Comment by pre-mRNA** [comment]
Yes, it would be incredible to split the alignment color from the modifications tag.  Thanks a lot.
